charts/send-letter-service/values.template.yaml

```diff
@@ -0,0 +1,40 @@
+java:
+  applicationPort: 8800
+  environment:
+    # logging
+    APPINSIGHTS_INSTRUMENTATIONKEY: "00000000-0000-0000-0000-000000000000"
+    # db - do not modify
+    LETTER_TRACKING_DB_NAME: hmcts
+    LETTER_TRACKING_DB_HOST: ${SERVICE_NAME}-postgresql
+    LETTER_TRACKING_DB_USER_NAME: hmcts
+    LETTER_TRACKING_DB_PASSWORD: hmcts
+    # auth
+    S2S_NAME: send_letter_tests
+    # ftp
+    FTP_HOSTNAME: "cmseft.services.xerox.com"
+    FTP_PORT: 22
+    FTP_FINGERPRINT: "SHA256:gYzreAtWAraVRFsOrcP9SPJq9atn7QxXh9pAauKud2U"
+    FTP_TARGET_FOLDER: "TO_XEROX"
```

Comment:
this should not be public ?

Reply:
i'm afraid everything is public: https://github.com/hmcts/send-letter-service/blob/b1cef96bac967074c11660ddb5d140a388425387/infrastructure/variables.tf#L77-L99